gravitee-apim-console-webui/src/management/api/api-traffic-v4/analytics/components/api-analytics-empty/api-analytics-empty.component.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, EventEmitter, Output } from '@angular/core';
+import { MatButton } from '@angular/material/button';
+import { MatCard, MatCardContent } from '@angular/material/card';
+import { MatIcon } from '@angular/material/icon';
+
+@Component({
+  selector: 'api-analytics-empty',
+  standalone: true,
+  imports: [MatButton, MatCard, MatCardContent, MatIcon],
```

Comment:
Shouldn't it be GioIconsModule ?

Reply:
hmm it has been done automatically, I wil ltake a look

Reply:
I do not need it with the use of the new component